client/web/src/open-in-editor/OpenInEditorPopover.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import { mdiClose } from '@mdi/js'
+
+import { Button, Icon } from '@sourcegraph/wildcard/out/src'
```

Comment:
```suggestion
import { Button, Icon } from '@sourcegraph/wildcard'
```

@vdavid, this change fixes the issue reported [here](https://sourcegraph.slack.com/archives/C01LTKUHRL3/p1662021068019189) for me locally.